cassandra/src/test/scala/filodb.cassandra/columnstore/CassandraColumnStoreSpec.scala

```diff
@@ -18,13 +18,15 @@ import filodb.memory.format.ZeroCopyUTF8String._
 import filodb.core.metadata.{Dataset, Schemas}
 import filodb.core.store.{ChunkSet, ChunkSetInfo}
 import filodb.core.store.ColumnStoreSpec
+import filodb.cassandra.DefaultFiloSessionProvider
 import filodb.cassandra.metastore.CassandraMetaStore
 
 class CassandraColumnStoreSpec extends ColumnStoreSpec {
   import NamesTestData._
 
-  lazy val colStore = new CassandraColumnStore(config, s)
-  lazy val metaStore = new CassandraMetaStore(config.getConfig("cassandra"))
+  lazy val session = new DefaultFiloSessionProvider(config.getConfig("cassandra")).session
+  lazy val colStore = new CassandraColumnStore(config, s, session)
```

Comment:
I think maybe this test should just use the `AllTablesTest` trait and avoid this duplication.

Reply:
This ends up creating a bunch of new compilation conflicts. Refactoring the tests is a bit more work and out of scope.